demisto_sdk/commands/format/tests/formatting_js_test.py

```diff
@@ -0,0 +1,33 @@
+import os
+import pytest
+import shutil
+
+from demisto_sdk.tests.constants_test import SOURCE_FORMAT_INCIDENTFIELD_COPY, DESTINATION_FORMAT_INCIDENTFIELD_COPY, \
+    SOURCE_FORMAT_INCIDENTTYPE_COPY, DESTINATION_FORMAT_INCIDENTTYPE_COPY, SOURCE_FORMAT_INDICATORFIELD_COPY, \
+    DESTINATION_FORMAT_INDICATORFIELD_COPY, SOURCE_FORMAT_INDICATORTYPE_COPY, DESTINATION_FORMAT_INDICATORTYPE_COPY, \
+    SOURCE_FORMAT_LAYOUT_COPY, DESTINATION_FORMAT_LAYOUT_COPY, SOURCE_FORMAT_DASHBOARD_COPY, \
+    DESTINATION_FORMAT_DASHBOARD_COPY, INCIDENTFIELD_PATH, DASHBOARD_PATH, LAYOUT_PATH, INCIDENTTYPE_PATH,\
+    INDICATORFIELD_PATH, INDICATORTYPE_PATH
+
+from demisto_sdk.commands.format.format_module import format_manager
+
+
+class TestFormattingJson:
```

Comment:
if not a good reason for the class, you can rm.

Reply:
also, let's rename the file to `formatting_json_test` as we refer internally usually js to javascript.